genesis/bin/genesis_playback.rs

```diff
@@ -232,26 +232,23 @@ fn main() {
             });
 
             // Push to storage.
-            let mut request = storage::ApplyRequest::new();
-            request.set_namespace(runtime_id.as_ref().to_vec());
-            request.set_src_round(0);
-            request.set_src_root(root.as_ref().to_vec());
-            request.set_dst_round(0);
-            request.set_dst_root(state_root.as_ref().to_vec());
-            request.set_log(
-                write_log
-                    .into_iter()
-                    .map(|entry| {
-                        let mut request = storage::LogEntry::new();
-                        request.set_key(entry.key);
-                        request.set_value(entry.value);
-
-                        request
-                    })
-                    .collect(),
-            );
-
-            storage.apply(&request).expect("storage apply must succeed");
+            storage
+                .apply(
+                    &storage::ApplyRequest {
+                        namespace: runtime_id,
+                        src_round: 0,
+                        src_root: root,
+                        dst_round: 0,
+                        dst_root: state_root,
+                        writelog: write_log
+                            .into_iter()
```

Comment:
I think that you can use the non-moving `.iter()` since you're just taking refs. Clippy will likely complain about this, if you'd notice it among the ~1000 other things